src/data/markdown/translated-guides/en/02 Using k6/05 Options.md

```diff
@@ -996,20 +996,20 @@ $ K6_STAGES="5s:10,5m:20,10s:5" k6 run ~/script.js
 ```
 
 ```bash
-c://k6> k6 run --stage 5s:10,5m:20,10s:5 ~/script.js
+C:\k6> k6 run --stage 5s:10,5m:20,10s:5 ~/script.js
 
 # or...
 
-c://k6> set "K6_STAGES=5s:10,5m:20,10s:5" && k6 run ~/script.js
+C:\k6> set "K6_STAGES=5s:10,5m:20,10s:5" && k6 run ~/script.js
```

Comment:
```suggestion
C:\k6> set "K6_STAGES=5s:10,5m:20,10s:5" && k6 run script.js
```

Reply:
This is found all over the place; not sure why it was added in the first place when it seems we are expected to be in a special `k6` folder. Removed.